charts/alfresco-repository/templates/configmap.yaml

```diff
@@ -9,10 +9,10 @@ metadata:
 data:
   CATALINA_OPTS: >-
     -Ddeployment.method=HELM_CHART
-    -Ddb.url=jdbc:{{ .Values.db.url }}
+    -Ddb.url=jdbc:{{ .Values.configuration.db.url }}
     -Ddb.username=${DATABASE_USERNAME}
     -Ddb.password=${DATABASE_PASSWORD}
-    -Ddb.driver={{ include "alfresco-repository.db.driver" .Values.db }}
+    -Ddb.driver={{ include "alfresco-repository.db.driver" .Values.configuration.db }}
```

Comment:
```suggestion
    -Ddb.driver={{ include "alfresco-repository.db.driver" .Values.configuration.db.driver }}
```

add a test

Reply:
Here I'm passing the context to the named template so I don't want to pass the driver.
There is already a test for each database type in `tests/config_test.yaml`
https://github.com/Alfresco/alfresco-helm-charts/blob/357447463b465ffd1144f0ad16a7ff65256fdb0a/charts/alfresco-repository/tests/configmap_test.yaml#L49

Reply:
oh yeah, looked like a mistype before reading the named template 😀